docs/src/modules/components/AppContent.js

```diff
@@ -15,10 +15,12 @@ const styles = theme => ({
       maxWidth: 'calc(100% - 175px)',
     },
     [theme.breakpoints.up('lg')]: {
-      paddingLeft: theme.spacing(1),
       paddingRight: theme.spacing(5),
       maxWidth: 'calc(100% - 240px - 175px)',
     },
+    '&:focus': {
+      outline: 'none',
```

Comment:
Should we target the `#main-content` id with a global style instead? We can still see the outline on the homepage.

![Capture d’écran 2019-04-21 à 17 11 54](https://user-images.githubusercontent.com/3165635/56471947-bb131380-6458-11e9-9ea4-215137884b69.png)